pkg/features/antrea_features_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package features
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/component-base/featuregate"
+)
+
+func TestSupportedOnWindows(t *testing.T) {
+	tcs := []struct {
+		name     string
+		feature  featuregate.Feature
+		expected bool
+	}{
+		{
+			name:     "Feature Supported on Windows Node",
+			feature:  AntreaPolicy,
+			expected: true,
+		},
+		{
+			name:     "Feature Unsupported on Windows Node",
+			feature:  Egress,
+			expected: false,
+		},
+		{
+			name:     "Feature does not exist",
+			feature:  "Unsupported",
+			expected: false,
+		},
+	}
+
+	for _, tt := range tcs {
+		t.Run(tt.name, func(t *testing.T) {
+			actual := SupportedOnWindows(tt.feature)
+			assert.Equal(t, tt.expected, actual)
+		})
+	}
+}
+
+func TestSupportedOnExternalNode(t *testing.T) {
+	tcs := []struct {
+		name     string
+		feature  featuregate.Feature
+		expected bool
+	}{
+		{
+			name:     "Feature Supported on External Node",
+			feature:  ExternalNode,
+			expected: true,
+		},
+		{
+			name:     "Feature Unsupported on External Node",
```

Comment:
ditto

Reply:
Btw, could you fix a typo here: https://github.com/antrea-io/antrea/blob/main/pkg/features/antrea_features.go#L211
replace `a external Node` to `an external Node`. thanks.

Reply:
fixed